src/audio/module_adapter/module/volume/volume_generic_with_peakvol.c

```diff
@@ -87,7 +87,7 @@ static void vol_s24_to_s24(struct processing_module *mod, struct input_stream_bu
 				y0[i] = vol_mult_s24_to_s24(x0[i], vol);
 				tmp = MAX(abs(x0[i]), tmp);
 			}
-			tmp = tmp << attenuation;
+			tmp = tmp << (attenuation + 8);
```

Comment:
Why not define a constant for 8 bit shift, e.g. `#define PEAK_METER_24S_32C_ADJUST 8`?

Reply:
thanks, done.